api.go

```diff
@@ -111,3 +112,13 @@ func (h *Hasher) Sum(b []byte) []byte {
 	h.h.finalize(tmp)
 	return append(b, tmp...)
 }
+
+// XOF returns an io.Reader containing 2^64 bytes of lazily generated hash output.
+//
+// It returns an io.ReadSeeker, but Seek is not implemented yet. This is to avoid
+// having to change the exported interface in the future.
+func (h *Hasher) XOF() io.ReadSeeker {
```

Comment:
Consider returning a concrete type that just implements io.ReadSeeker. That would let many callers avoid the interface indirection.

Reply:
I agree.